resources/factions/russia_1975.json

```diff
@@ -7,7 +7,9 @@
     "ru_RU"
   ],
   "aircrafts": [
+    "IL-76MD",
     "Mi-24V Hind-E",
+    "Mi-24P Hind-F",
```

Comment:
This was intentional (as was adding the new "Russia 1975 Mi-24P" faction). This aircraft wasn't used in 1975. The difference is historical accuracy for opfor vs flyability for players, not mod support.

Reply:
https://github.com/dcs-liberation/dcs_liberation/pull/1267

Reply:
ok it really does seem weird to keep them both since only one is flyable but ok, i'm assuming your adding it back in and I don't need to add it back in. Should I audit our factions for accuracy then, cause I didn't do that

Reply:
> Should I audit our factions for accuracy then, cause I didn't do that

If you want. The author of the PR did but I don't think it's urgent.